### PR TITLE
feat: SilentHotbar module

### LINKED ISSUE #4204: [FEATURE] SilentHotbar module
### LiquidBounce Branch

Nextgen

### Describe your feature request.

A visual module called Silent Hotbar would be nice to have. It should be able to let you customise whether your displayed hand is showing the silent item, or is showing the item you are currently holding.

### Additional context

For example, it would let you visually hold a sword whilst scaffolding.

### PR DESCRIPTION
**Pull request by ccetl**
Closes #4204

### CODE REVIEW

**Comment by modelpackagespaceingrootjujummail** [comment]
i came to this

**Comment by liquidsquid1** [comment]
> Closes #4204

thank you 🙏 i shall check in the morning if this is compatible with noswing and other stuff

File: src/main/java/net/ccbluex/liquidbounce/injection/mixins/minecraft/item/MixinHeldItemRenderer.java [hunk 1]
```diff
@@ -170,4 +173,13 @@ private void transformLegacyBlockAnimations(AbstractClientPlayerEntity player, f
         }
     }
 
+    @Redirect(method = "updateHeldItems", at = @At(value = "INVOKE", target = "Lnet/minecraft/client/network/ClientPlayerEntity;getMainHandStack()Lnet/minecraft/item/ItemStack;"))
```
Comment (1zun4) [inline src/main/java/net/ccbluex/liquidbounce/injection/mixins/minecraft/item/MixinHeldItemRenderer.java]: Please do not use `@Redirect`

Comment (ccetl) [reply src/main/java/net/ccbluex/liquidbounce/injection/mixins/minecraft/item/MixinHeldItemRenderer.java]: I doubt any mod will create a mixin to the exact same point but sure I'll change it to `ModifyExpressionValue`.

**Pull request merged**